config/connection.js

```diff
@@ -2,13 +2,14 @@
 
 // Set up MySQL connection
 const mysql = require("mysql");
+const Sequelize = require("sequelize");
 
 let connection = mysql.createConnection({
   host: "localhost",
   port: 3306,
   user: "root",
   password: "password",
-  database: "recycleThis_db"
+  database: "recyclethis_db"
```

Comment:
recycleThis_db

Reply:
Oh yea, sequelize didn't like the camelcase so I had to change everything to lowercase to get it to work